src/endpoint.rs

```diff
@@ -66,6 +66,12 @@ where
     }
 }
 
+impl<State> Endpoint<State> for &'static str {
```

Comment:
Why's this in the same change?

Reply:
See note above ☝️

Reply:
It'd be nice to have it in a separate commit

Reply:
Entirely backed it out, wasn't used anywhere